src/migrations/20231012082537-message-banners.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+exports.up = function (db, cb) {
+    db.runSql(
+        `
+        CREATE TABLE IF NOT EXISTS message_banners
+            (
+                id SERIAL PRIMARY KEY NOT NULL,
+                enabled BOOLEAN DEFAULT true NOT NULL,
+                message TEXT NOT NULL,
+                variant TEXT,
```

Comment:
Is this meant as type of message? Like warning, info, error etc?

Reply:
Yep, exactly! Do you think we should have a different type here?

Reply:
I was just pondering the use case, as the name was so close to a part of our domain (variants and features/toggles). We could call it message_type or banner_type? Just suggestions though

Reply:
That's a fair point 🤔 I think I prefer to proceed like this to ensure existing external message banner payloads don't break, but it's something I'll take into account and get back to if we identify some confusion.